HW8/marian_hrychak/task_3/main.py

```
@@ -0,0 +1,18 @@
+from math_utils import *
```

Ruff (0.8.2)

`from math_utils import *` used; unable to detect undefined names

(F403)